assemblyline/odm/models/submission.py

```diff
@@ -114,46 +120,49 @@ def create_filescore_key(self, sha256, services: list = None):
+""")
 class Verdict(odm.Model):
-    malicious = odm.List(odm.Keyword(), default=[], description="List of user that thinks this submission is malicious")
+    malicious = odm.List(odm.Keyword(), default=[], description="A list of usernames representing users who have judged the submission to be malicious.")
     non_malicious = odm.List(
         odm.Keyword(),
         default=[],
-        description="List of user that thinks this submission is non-malicious")
+        description="A list of usernames representing users who have judged the submission to be non-malicious.")
 
 
-@odm.model(index=True, store=True, description="Model of Submission")
+@odm.model(index=True, store=True, description="""A Submission in Assemblyline is a critical entity that encapsulates the analysis process and data for a file or collection of files submitted to determine the presence of malware. It contains a wealth of attributes such as file details, parameters for analysis, and the outcome of the scan. Familiarity with the Submission model is essential for users who need to construct precise Lucene search queries. This understanding enables them to effectively navigate and query the Assemblyline system to obtain detailed information on submissions, streamline their search efforts, and efficiently access the desired analysis results.
+""")
 class Submission(odm.Model):
-    archive_ts = odm.Optional(odm.Date(description="Time at which the submission was archived", ai=False))
-    archived = odm.Boolean(default=False, description="Document is present in the malware archive", ai=False)
-    classification = odm.Classification(description="Classification of the submission")
-    error_count = odm.Integer(description="Total number of errors in the submission", ai=False)
-    errors: list[str] = odm.List(odm.Keyword(), store=False, description="List of error keys", ai=False)
-    expiry_ts = odm.Optional(odm.Date(store=False), description="Expiry timestamp", ai=False)
-    file_count = odm.Integer(description="Total number of files in the submission", ai=False)
-    files: list[File] = odm.List(odm.Compound(File), description="List of files that were originally submitted")
-    max_score = odm.Integer(description="Maximum score of all the files in the scan")
-    metadata = odm.FlattenedObject(store=False, description="Metadata associated to the submission")
-    params: SubmissionParams = odm.Compound(SubmissionParams, description="Submission parameter details", ai=False)
-    results: list[str] = odm.List(odm.Keyword(), store=False, description="List of result keys", ai=False)
-    sid: str = odm.UUID(copyto="__text__", description="Submission ID")
-    state = odm.Enum(values=SUBMISSION_STATES, description="Status of the submission", ai=False)
+    archive_ts = odm.Optional(odm.Date(ai=False), description="An optional timestamp indicating when the submission was archived.")
```

Comment:
Why did you move the ai=False on the odm.Date for this case, but not for expiry_ts? Not saying it should, just that now we have two different behaviours?

Reply:
I did not move it, it was already in the `odm.Date`.  I moved `description` outside so that the Python code generating the docs would pick it up.  But I agree that it is inconsistant with the rest, and I did not fix it to avoid breaking other things.

Reply:
You're right. I misread the parenthesis. Probably a typo that would need to be confirmed before changing.